site/content/zh-CN/docs/tasks/run/rayservices.md

```diff
@@ -1,48 +1,52 @@
 ---
-title: "Run A RayService"
-linkTitle: "RayServices"
+title: "运行 RayService"
+linkTitle: "Ray 服务"
 date: 2025-06-30
 weight: 6
 description: >
-  Run a RayService with Kueue.
+  在 Kueue 上运行 RayService 的指南。
 ---
 
-This page shows how to leverage Kueue's scheduling and resource management capabilities when running [RayService](https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html).
+本页演示如何利用 Kueue 的调度与资源管理能力运行 [RayService](https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html) 。
 
-Kueue manages the RayService through the RayCluster created for it. Therefore, RayService needs the label of `kueue.x-k8s.io/queue-name: user-queue` and this label is propagated to the relevant RayCluster to trigger Kueue's management.
+Kueue 通过为 RayService 创建的 RayCluster 来管理 RayService。
+因此，RayService 需要在 metadata.labels 中包含 `kueue.x-k8s.io/queue-name: user-queue` 标签，该标签会被传递到相应的 RayCluster，以接收来自 Kueue 的管理。
```

Comment:
```suggestion
因此，RayService 需要在 metadata.labels 中包含 `kueue.x-k8s.io/queue-name: user-queue` 标签，
该标签会被传递到相应的 RayCluster，以触发 Kueue 的管理。
```

Reply:
Please use Copy + Paste. Do not identify the differences with your eyes, which may ignore somthing you don't know or not aware of.